### DISCUSSION TITLE
Regarding task composer plugins in planning/src/factories

### DISCUSSION THREAD

**Discussion by ragesh88** [tesseract-robotics/tesseract_planning#566] [General]
Hi,

Where can I find the description on the difference between the different pipelines: DescartesDPipeline, DescartesDTask, DescartesFPipeline, DescartesFTask

Thanks

**Comment by marip8** [comment]
The letters F and D stand for float and double precision, respectively. The idea is that you should be able to support larger planning problems (i.e., finer discretization) in memory if you use a floats instead of doubles to represent joint values.

The task is just the Descartes motion planner itself, and the pipeline is a sequence of tasks including the Descartes planner: essentially planning checks -> Descartes -> fine resolution collision check -> time parameterization

**Comment by ragesh88** [comment]
Okay thanks
